scripts/apps/ingest/views/settings/ingest-routing-general.html

```diff
@@ -1,70 +1,74 @@
-<div class="field">
-    <label translate>Rule name</label>
-    <input type="text" class="line-input" placeholder="{{ 'Rule name'|translate }}" ng-model="rule.name" required>
-</div>
+<fieldset class="">
+    <div class="form-block">
+        <h3 class="form-block__title" translate>Content Filter</h3>
+        
+        <div class="list-row list-row--dotted list-row--flex" ng-if="rule.filter">
```

Comment:
imo we're using way too much markup all over the place, we should have directives in views and only use classes there imo. this could be some:
```
<div sd-list-row data-label="{{ 'Name' | translate }}" dotted>{{ rule.filterName }}</div>
```
otherwise we won't make it consistent and it's hard to maintain this